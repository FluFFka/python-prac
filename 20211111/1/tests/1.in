a = OrderedComplex(1, 2)
b = OrderedComplex(1 + 2j)
c = OrderedComplexMul(2.25-5j)
d = OrderedComplexMul(3,4.4568879)
print(a < a)
print(a < b)
print(a < c)
print(a < d)
print(b < a)
print(b < b)
print(b < c)
print(b < d)
print(c < a)
print(c < b)
print(c < c)
print(c < d)
print(d < a)
print(d < b)
print(d < c)
print(d < d)
print(c@c)
print(c@d)
print(d@c)
print(d@d)

