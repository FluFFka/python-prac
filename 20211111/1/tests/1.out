False
False
False
True
False
False
False
True
False
False
False
True
False
False
False
False
30.0625
-15.534439499999998
-15.534439499999998
28.86384975316641
